src/home_robot/home_robot/utils/point_cloud.py

```diff
@@ -44,7 +44,9 @@ def show_pcd(
     save: str = None,
     grasps: list = None,
 ):
-    geoms = create_visualization_geometries(pcd=pcd, orig=orig, R=R, grasps=grasps)
+    geoms = create_visualization_geometries(
+        pcd=pcd, orig=orig, R=R, grasps=grasps, size=0.1
```

Comment:
why is size hard coded? should we pass it in as a param?

Reply:
I'll add a default and pass it in as params. It is *sorely* needed though, for some reason the open3d coordinate axis geoms are humongous compared to our point-clouds.